shell.nix

```diff
@@ -59,15 +59,6 @@ let
 
     CARDANO_NODE_CONFIGS = pkgs.cardano-node-deployments;
 
-    # If any build input has bash completions, add it to the search
```

Comment:
Ooh, what makes this not necessary anymore?

Reply:
I put a link in the commit message to this: NixOS/nixpkgs#103501

Reply:
Thank you :+1: